### PR TITLE
ci: prepare assets for release

### PR DESCRIPTION
**Pull request by soc1c**
Creates artifacts on CircleCI we can use for releases: 

- `geth-classic-v6.0.1-16-g7f356f2-linux.tar.gz`
- `geth-classic-v6.0.1-16-g7f356f2-linux.sha256`

### CODE REVIEW

**Comment by soc1c** [comment]
We have binaries!

Screenshot_2019-06-17 Continuous Integration and Deployment

**Review by noot — APPROVED** [review]

**Review by austinabell — APPROVED** [review]

**Review by GregTheGreek — DISMISSED** [review]
My only comment would be that technically it works for osx as well as linux

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,8 +10,30 @@ jobs:
...
+            VERSION=$(bin/geth version | grep -i version | head -n1 | awk '{print $2}')
+            cp -rv ./bin ./geth-classic-${VERSION}-linux
+            tar -zcvf geth-classic-${VERSION}-linux.tar.gz geth-classic-${VERSION}-linux
+            cp -v ./geth-classic-${VERSION}-linux.tar.gz ./artifacts/
+            pushd ./artifacts
+            sha256sum ./geth-classic-${VERSION}-linux.tar.gz > ./geth-classic-${VERSION}-linux.sha256
+            popd
+      - store_artifacts:
+          path: ~/go-ethereum/artifacts
       - save_cache:
```
Comment (GregTheGreek) [inline .circleci/config.yml:37]:
```suggestion
      - run:
          name: "Publish Release on GitHub"
          command: |
            VERSION=$(bin/geth version | grep -i version | head -n1 | awk '{print $2}')
            go get github.com/tcnksm/ghr
            ghr -t ${GITHUB_TOKEN} -u eth-classic -r go-ethereum -c ${CIRCLE_SHA1} ./artifacts/
      - save_cache:
```

**Review by GregTheGreek — CHANGES_REQUESTED** [review]
This should upload the artifacts for us :)

File: .circleci/config.yml (hunk 1 above)
Comment (GregTheGreek) [reply .circleci/config.yml:37]: hold up, need to add tag specific releases

**Review by GregTheGreek — APPROVED** [review]

**Comment by GregTheGreek** [comment]
I'll add auto deploy in a seperate PR :) Good to go @soc1c

**Pull request merged**